src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,77 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { color } from '../../styles';
+import GymLogo from '../GymLogo';
+
+interface Props {
+  children: React.ReactNode;
+  /**
+   * Position of navigation links (or other content) inside the `NavBar`
+   * Valid values are "left", "right", and "center"
+   */
+  justifyContent?: 'left' | 'right' | 'center';
+  /**
+   * Set the backgound color of the `NavBar`
+   */
+  backgroundColor: string;
+  /** Height Override */
+  height?: string;
+  /** Display Gymnasium Logo
+   * To display, use `left` or `right`
+   * To hide, don't specify this prop or use `no`
+   */
+  displayLogo?: 'left' | 'right' | 'no';
```

Comment:
This likely can just be a `bool` with t/f values 👍

Reply:
@mbifulco OK, I split this into two props, one of them is boolean and the other for the position.